parlai/utils/bpe.py

```diff
@@ -219,6 +229,17 @@ def sync_with_dict(self, dict_agent):
             agent with which we are syncing the dictionary
         """
 
+    def add_special_tokens(self, dict_agent, special_tokens: List[str]):
+        """
+        Add special tokens to the tokenizer.
+
+        These tokens are treated as discreet tokens, and prioritized over the BPE
```

Comment:
nit: discrete ? but i'm also not sure what you're trying to say here

Reply:
ya bad english